documentdata.py

```diff
@@ -30,6 +33,55 @@ def __init__(self, name: str, path: str, type: str, lang: str, size: int, create
         self.createdAt = createdAt
         self.tags = tags
 
+    @classmethod
+    def documentdata_from_result(cls, result: dict):
```

Comment:
just call it `from_result` or `create_from_result`, but I think the first one is understandable IMHO